gensrc/script/doris_builtins_functions.py

```diff
@@ -1036,13 +1036,13 @@
     [['timediff'], 'TIMEV2', ['DATETIMEV2', 'DATETIMEV2'], ''],
 
     [['datediff'], 'INT', ['DATETIMEV2', 'DATEV2'], ''],
-    [['timediff'], 'TIMEV2', ['DATETIMEV2', 'DATEV2'], ''],
+    [['timediff'], 'TIME', ['DATETIMEV2', 'DATEV2'], ''],
```

Comment:
why use time for datetimev2 and datev2? do not need microsecond diff?

Reply:
I think that timev2 should only be calculated when both types have microseconds.

Reply:
If want to calculate microseconds,  cast "datev2" to "datetimev2"?

Reply:
if the diff include second, it should include microsecond too